packages/server/lib/controllers/v1/getEnvJs.ts

```diff
@@ -1,19 +1,22 @@
+import { envs } from '@nangohq/logs';
 import { basePublicUrl, baseUrl, connectUrl, flagHasAuth, flagHasManagedAuth, flagHasScripts, flagHasSlack, isCloud } from '@nangohq/utils';
+
 import { asyncWrapper } from '../../utils/asyncWrapper.js';
+
 import type { WindowEnv } from '@nangohq/types';
-import { envs } from '@nangohq/logs';
 
 export const getEnvJs = asyncWrapper<any, any>((_, res) => {
     const configObject: WindowEnv = {
         apiUrl: baseUrl,
         publicUrl: basePublicUrl,
         connectUrl: connectUrl,
-        publicSentryKey: process.env['PUBLIC_SENTRY_KEY'] || '',
-        publicPosthogKey: process.env['PUBLIC_POSTHOG_KEY'] || '',
-        publicPosthogHost: process.env['PUBLIC_POSTHOG_HOST'] || '',
-        publicLogoDevKey: process.env['PUBLIC_LOGODEV_KEY'] || '',
-        publicKoalaApiUrl: process.env['PUBLIC_KOALA_API_URL'] || '',
-        publicKoalaCdnUrl: process.env['PUBLIC_KOALA_CDN_URL'] || '',
+        gitHash: envs.GIT_HASH,
```

Comment:
how is the env var set? would it make more sense to build a `version.json` static file that server could serve?

Reply:
it's set in the docker image.

getEnvJs.ts is already a "static file" kind of, I don't see a big need to compile a file at build time but yes it's an alternative

Reply:
can we have a comment stating when GIT_HASH is being set?

Reply:
It's already written in the parse 
https://github.com/NangoHQ/nango/blob/08dd308e52546014b0313575dd0e129a79c4a794/packages/utils/lib/environment/parse.ts#L15-L16